src/Command/RoleCreatorCommand.php

```diff
@@ -209,4 +210,14 @@ private function findDocWithName(string $name, array $docTypes): ?DocType
 
         return null;
     }
+
+    private function applyPerspectives(Role $role, array $roleProperties): void
+    {
+        if (!key_exists("perspectives", $roleProperties)) {
+            $role->setPerspectives([]);
+            return;
+        }
+
+        $role->setPerspectives($roleProperties["perspectives"]);
```

Comment:
I believe the null coalescing operator works with array accessing now, so you could simplify this to:
```suggestion
        $role->setPerspectives($roleProperties["perspectives"] ?? []);
```
and if "perspectives" isn't in `$roleProperties` it won't actually throw an exception. Also this only works so long as you trust that `$roleProperties["perspectives"]` is non null

Reply:
Nice, I just tried it out and it looks like that should work.

Reply:
Nice I just tried that out, and it looks like that works as expected